app/controllers/idv/in_person/state_id_controller.rb

```diff
@@ -113,6 +115,10 @@ def analytics_arguments
           merge(extra_analytics_properties)
```

Comment:
this `analytics_arguments` method also seems like a good place to add in the DOB param here, it's already merged into the result passed to analytics?

Reply:
I think this is a good suggestion.  I agree with trying to add the birth year here or in the other suggested spot, rather than creating the new `form_analytics_properties` method.